### ISSUE TITLE
Warp Pack Organic Teleportation

### ISSUE THREAD

**Issue by aleksix** [Aurorastation/Aurora.3#8502]
Game ID: b6l-b0T8

Warp packs can teleport organics that are put inside body bags, even from the inside, which is supposedly not an intended feature. Probably possible with stasis-bags and oxygen bubbles as well.

**Comment by SleepyWolfe** [comment]
and probably lockers

**Comment by aleksix** [comment]
> 
> 
> and probably lockers

I don't think so. If I remember correctly, lockers get opened when you click them with items from the inside if they are unlocked, and you can't lock the locker from the outside.
